strawberryfields/ops.py

```diff
@@ -1712,31 +1714,29 @@ class GaussianTransform(Decomposition):
     .. math:: O_i = \begin{bmatrix}X&-Y\\Y&X\end{bmatrix}
 
     Args:
-        S (array): a :math:`2N\times 2N` symplectic matrix describing the Gaussian transformation.
+        S (array[float]): a :math:`2N\times 2N` symplectic matrix describing the Gaussian transformation.
         vacuum (bool): set to True if acting on a vacuum state. In this case, :math:`O_2 V O_2^T = I`,
             and the unitary associated with orthogonal symplectic :math:`O_2` will be ignored.
         tol (float): the tolerance used when checking if the matrix is symplectic:
             :math:`|S^T\Omega S-\Omega| \leq` tol
     """
-    ns = None
```

Comment:
It doesn't break the Program by using an operation with `ns` not defined?

Reply:
Oops, you're right.